components/TodoItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { StyleSheet, Text, Button, TouchableOpacity } from 'react-native'
+
+export default class TodoItem extends React.Component {
+    constructor (props) {
+        super(props)
+    }
+
+render () {
+    const todoItem = this.props.todoItem
+
+    return (
+        <TouchableOpacity 
+        style={styles.todoItem}
+        onPress={() => this.props.toggleDone()}
+        >
+            <Text style={(todoItem.done) ? {color: '#AAAAAA'} : {color: '#313131'}}>
+                { todoItem.title }
+            </Text>
+
+            <Button
+            title="Remove"
+            color={(todoItem.done) ? 'rgba(200, 0, 0, 0.5)' : 'rgba(255, 0, 0, 1)' }
+            onPress={() => this.props.removeTodo()}
+            />
+        </TouchableOpacity>
+    )

```

Comment:
The remove button doesn't show. Try putting the button outside/below the TouachableOpacity